.circleci/config.yml

```diff
@@ -354,6 +379,9 @@ workflows:
                 - develop
     jobs:
       - dependencies_unix
-      - test_nightly:
+      - test_nightly_chrome:
```

Comment:
How will we know if/when this fails? Is there someone responsible for checking CI logs daily?

Reply:
Yeah, those get kicked out to our #attest-build channel on Slack.